lib/screenplay_web/router.ex

```diff
@@ -35,32 +35,32 @@ defmodule ScreenplayWeb.Router do
   # Load balancer health check
   # Exempt from auth checks and SSL redirects
   scope "/", ScreenplayWeb do
-    get "/_health", HealthController, :index
+    get("/_health", HealthController, :index)
   end
 
   scope "/", ScreenplayWeb.OutfrontTakeoverTool do
-    pipe_through [
+    pipe_through([
       :redirect_prod_http,
       :browser,
       :auth,
       :ensure_auth,
       :ensure_screenplay_admin_group
-    ]
+    ])
 
     get("/", PageController, :takeover_redirect)
     get("/emergency-takeover", PageController, :index)
   end
 
   scope "/", ScreenplayWeb do
-    pipe_through [:redirect_prod_http, :browser, :auth, :ensure_auth]
+    pipe_through([:redirect_prod_http, :browser, :auth, :ensure_auth])
 
     get("/dashboard", DashboardController, :index)
-    get("/alerts", AlertsController, :index)
+    get("/alerts/*id", AlertsController, :index)
```

Comment:
Do we need this `*/id` in the route definition here? Actually I'm slightly confused because the index function in `AlertsController` doesn't seem to do anything 🤔 at least in this branch?